examples/image.rs

```diff
@@ -0,0 +1,23 @@
+use cherry::widget::{image::Image, Widget};
+use embedded_graphics::{
+    pixelcolor::Rgb888,
+    prelude::{Point, Size},
+};
+use embedded_graphics_simulator::{OutputSettingsBuilder, SimulatorDisplay, Window};
+use std::convert::Infallible;
+use tinybmp::Bmp;
+
+fn main() -> Result<(), Infallible> {
+    let mut display: SimulatorDisplay<Rgb888> = SimulatorDisplay::new(Size::new(300, 300));
+
+    let image_data = include_bytes!("./images/cherry.bmp");
```

Comment:
This inlines this file into the code as an array of bytes.

Reply:
Is `.bmp` an expected image format for embedded?

Reply:
I'm not sure what's normal in non-rust situations, but I think it's easy to support because it's a pretty "dumb" format: after a short headers, it's just the raw pixel data (no compression or anything). So inefficient space-wise, I guess, but low cost for the CPU?